yt_dlp/extractor/googledrive.py

```diff
@@ -56,24 +57,20 @@ class GoogleDriveIE(InfoExtractor):
     }, {
         'url': 'https://drive.usercontent.google.com/download?id=0ByeS4oOUV-49Zzh4R1J6R09zazQ',
         'only_matching': True,
+    }, {
+        'url': 'https://drive.usercontent.google.com/download?id=0ByeS4oOUV-49Zzh4R1J6R09zazQ',
+        'only_matching': True,
+    }, {
+        # mp3 (format 50) tests
+        # Royalty Free music from 1922
+        # My Buddy by Henry Burr; Gus Kahn; Walter Donaldson
+        # https://archive.org/details/78_my-buddy_henry-burr-gus-kahn-walter-donaldson_gbia0015627a/My+Buddy+-+Henry+Burr+-+Gus+Kahn+-+Walter+Donaldson.flac
+        'url': 'https://drive.usercontent.google.com/download?id=1IP0o8dHcQrIHGgVyp0Ofvx2cGfLzyO1x',
+        'only_matching': True,
```

Comment:
can we make this a full/proper test instead of `only_matching`?

Reply:
Yes, done.